### PR TITLE
Feature/syal 838

### PR DESCRIPTION
**Pull request by HieuLM-AVISPL**
Hi @MaksimR-AVISPL 
This is the implement for the qsys reflect V2
Please help me to review it. 
Thanks

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -198,7 +198,7 @@
       <groupId>org.snmp4j</groupId>
       <artifactId>snmp4j</artifactId>
       <scope>test</scope>
-      <version>3.5.1</version>
```
Comment (MaksimR-AVISPL) [inline pom.xml:201]: 3.5.1 is the correct version, is there a reason it is reverted?

File: src/main/java/com/avispl/symphony/dal/infrastructure/management/qsc/qsysreflect/QSysReflectCommunicator.java [hunk 2]
```diff
@@ -587,6 +701,38 @@ private void retrieveDevices() {
...
 			aggregatedDeviceList = new ArrayList<>(aggregatedDeviceProcessor.extractDevices(devices));
+			for (AggregatedDevice aggregatedDevice : aggregatedDeviceList) {
+				Map<String, String> stats = aggregatedDevice.getProperties();
+				stats.put(QSysReflectConstant.DEVICE_TYPE, QSysReflectConstant.CORE);
+				aggregatedDevice.setProperties(stats);
+			}
+		} catch (Exception e) {
+			String errorMessage = String.format("Aggregated Device Data Retrieval-Error: %s with cause: %s", e.getMessage(), e.getCause().getMessage());
+			deviceErrorMessagesList.add(errorMessage);
+			logger.error(errorMessage);
```
Comment (MaksimR-AVISPL) [inline src/main/java/com/avispl/symphony/dal/infrastructure/management/qsc/qsysreflect/QSysReflectCommunicator.java]: can you please add an original error here as well? Just not to swallow and lose it

File: src/main/java/com/avispl/symphony/dal/infrastructure/management/qsc/qsysreflect/QSysReflectCommunicator.java [hunk 3]
```diff
@@ -334,7 +429,7 @@ protected void internalInit() throws Exception {
 		apiToken = this.getPassword();
 		this.setBaseUri(QSysReflectConstant.QSYS_BASE_URL);
 		if (checkValidApiToken()) {
-			executorService = Executors.newSingleThreadExecutor();
+			executorService = Executors.newFixedThreadPool(8);
```
Comment (MaksimR-AVISPL) [inline src/main/java/com/avispl/symphony/dal/infrastructure/management/qsc/qsysreflect/QSysReflectCommunicator.java:436]: is this the most optimal number for the aggregator? Are there any API limitations on number of requests per second?

**Review by MaksimR-AVISPL — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (HieuLM-AVISPL) [reply pom.xml:201]: Hi @MaksimR-AVISPL information from Ivan and Kevin. we discussed earlier will downgrade version 2.8.9 for (test cope).
image

Comment (HieuLM-AVISPL) [reply pom.xml:201]: Hi @MaksimR-AVISPL Information from Ivan and Kevin. We discussed earlier will downgrade version 2.8.9 (test cope)
image

**Pull request merged**